spec/config/lint-on-fly-spec.coffee

```diff
@@ -0,0 +1,11 @@
+describe "The Lint on the Fly Configuration Option", ->
```

Comment:
Why have an individual file for each config and not:


```CoffeeScript
describe "config", ->
    describe "The Lint on the Fly Configuration Option", ->
        it "is `true` by default.", ->
    describe "The Inline Tooltips Configuration Option", ->
        it "is `true` by default.", ->
```

That seems to be what Jasmine is designed for.